cartoframes/viz/helpers/isolines_layer.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+
+from . import color_category_layer
+from ...data.dataset.dataset import Dataset
+
+
+def isolines_layer(source, value='data_range', **kwargs):
+    """Helper function for quickly creating an isolines color map.
+
+    Args:
+        source (:py:class:`cartoframes.data.Dataset>`, DataFrame or str): Dataset
+          or text representing a table or query associated with user account.
+        value (str): Column to symbolize by. By default is "data_range".
+        title (str, optional): Title of legend.
+        top (int, optional): Number of category for map. Default is 11. Values
+          can range from 1 to 16.
+        cat (str, optional): Category list. Must be a valid CARTO VL category
```

Comment:
As an improvement, I'd replace the `Must be a valid CARTO VL category` (both here and in the `color_category_layer` helper) with: `must be a valid list of categories` (?)